### PR TITLE
Holodeck update v2

### PR DESCRIPTION
**Pull request by Lokiamis**
#13672 
Redone to fix conflicts.

untitled

:cl:
 * rscadd: Holodeck updated

### CODE REVIEW

**Comment by Lokiamis** [comment]
@Probe1 @Intigracy 
great

**Comment by CptWad** [comment]
it complies just merge it now

**Comment by Ephx1** [comment]
AI HOLODECK IMPLEMENTATION _2017_

**Comment by AI-Is-Rogue** [comment]
@Ephx1 AIs get a physical form in the holodeck?

**Comment by Ephx1** [comment]
That's how it is on Bay anyways. It'd make being a hologram more fun at least, and makes sense in the holodeck.

**Comment by Yelier** [comment]
This is really cool. Merge this soon, I really look forward to using it.

**Comment by AI-Is-Rogue** [comment]
@Ephx1 Imagine triumvirate AIs each on different lawsets and having a tea party in the holodeck

File: code/game/area/Space Station 13 areas.dm [hunk 1]
```diff
@@ -1173,6 +1173,38 @@ proc/process_adminbus_teleport_locs()
 	name = "\improper Holodeck - Space"
 	jammed=SUPER_JAMMED
 
+/area/holodeck/source_firingrange
+	name = "\improper Holodeck - Firing Range"
+	jammed=SUPER_JAMMED
```
Comment (Probe1) [inline code/game/area/Space Station 13 areas.dm]: Put spaces on all of these.  Yes I know your eyes are rolling.  No I don't like saying it.

Comment (Probe1) [inline code/game/area/Space Station 13 areas.dm:1178]: I take it back the whole files style is shitty so I'm not gonna ask you to change it.

Comment (Lokiamis) [inline code/game/area/Space Station 13 areas.dm]: Do you mean space it out to jammed = SUPER_JAMMED? I don't mind doing it for the whole file.

**Comment by N3X15** [comment]
https://github.com/d3athrow/vgstation13/pull/13494 has Map Priority

**Comment by Lokiamis** [comment]
We could just break the Deff holodeck for a bit and pretend it was an accident. I haven't touched it yet.

untitled

**Comment by Shadowmech88** [comment]
Or we could pretend it *wasn't* an accident because Deff is a broken-down garbage station so it gets a matching holodeck.

**Comment by Lokiamis** [comment]
I like they way you think

**Comment by Lokiamis** [comment]
Merge whenever and I'll throw Deff up shortly after. It's done but I'm to inteligent to add it to this PR without causing conflicts.

**Pull request merged**